### PR TITLE
add new config uva-2 for U. Virginia locally digitized content

### PR DESCRIPTION
**Pull request by bwcormack**
@RvanB - Another to review when you have time. No rush. I can do the merge after. Thx!

### CODE REVIEW

File: uva-2.config [hunk 1]
```diff
@@ -0,0 +1,18 @@
+uva-2.bar_field = 955
+uva-2.bar_field_mult = 1 
+uva-2.bar_sfield = i
+uva-2.campus_code = uva
```
Comment (RvanB) [inline uva-2.config:4]: This doesn't conflict with the first uva config?

Comment (RvanB) [reply uva-2.config:4]: I guess my real question is, will uva-2 files go in a separate directory from the old uva?

I'm not clear if this config campus_code is the same as the "campus code" used in process_indata to apply the special sauce.

Comment (bwcormack) [reply uva-2.config:4]: The code in the config's campus_code is used to prefix the bib record's system ID, to form the "sdr" number. I believe the campus code used in process_indata is the 'name' of the stream, which would be uva-2. Should we double-check? The tests on dev have gone correctly.

Comment (RvanB) [reply uva-2.config:4]: Okay thanks! That answers my question, should be good to go.

**Review by RvanB — APPROVED** [review]

File: uva-2.config (hunk 1 above)
Comment (bwcormack) [reply uva-2.config:4]: Just a bit more - my notes in ticket 21371 indicate that process_indata defines a 'campus' variable according to the name of a directory. So I believe we're all set, thx!

**Pull request merged**